src/hotspot/share/utilities/debug.hpp

```diff
@@ -245,7 +245,7 @@ do {
 
 
 // types of VM error - originally in vmError.hpp
-enum VMErrorType {
+enum VMErrorType : unsigned int {
```

Comment:
Why 'unsigned int' when the other changes are `uint`?

Reply:
`uint` is not available in this header file, and I didn't want to add includes.